Themis/Models/Assessment.swift

```diff
@@ -72,7 +72,7 @@ struct AssessmentResult: Encodable {
     }
 }
 
-struct AssessmentFeedback: Identifiable {
+class AssessmentFeedback: Identifiable, Decodable {
```

Comment:
Hmm, I don't see problems with the changes in this file, but why are they necessary?

Reply:
The only change that is required is the switch from struct to class, all other changes in AssessmentFeedback are just adaptions that are necessary for the compiler.
The reason I switched to a class is that it's the most convenient way to make sure that the changes are directly published to the ListView since both views are then referencing the same object.

Reply:
Ok!